brainglobe_stitch/stitching_widget.py

```diff
@@ -212,6 +208,19 @@ def __init__(self, napari_viewer: Viewer):
         self.stitch_button.setEnabled(False)
         self.layout().addWidget(self.stitch_button)
 
+        self.fuse_option_widget = QWidget()
+        self.fuse_option_widget.setLayout(QFormLayout())
+        self.output_file_name_field = QLineEdit()
+        self.fuse_option_widget.layout().addRow(
+            "Output file name:", self.output_file_name_field
+        )
+
+        self.layout().addWidget(self.fuse_option_widget)
+
+        self.fuse_button = QPushButton("Fuse")
```

Comment:
ATM the fuse button is active when GUI is loaded. It might be nice to activate only when data is stitched.